src/Jackett.Common/Indexers/HDOlimpo.cs

```diff
@@ -159,49 +157,46 @@ private List<ReleaseInfo> ParseResponse(WebClientStringResult response, bool inc
                     if ((string)torrent["premium"] == "si")
                     {
                         if (includePremium)
-                            release.Title += " [PREMIUM]";
+                            title += " [PREMIUM]";
                         else
                             continue;
                     }
 
-                    release.Comments = new Uri(CommentsUrl + (string)torrent["id"]);
-                    release.Guid = release.Comments;
-
-                    release.PublishDate = DateTime.Now;
+                    var guid = new Uri(CommentsUrl + (string)torrent["id"]);
+                    var publishDate = DateTime.Now;
                     if (torrent["created_at"] != null)
-                        release.PublishDate = DateTime.Parse((string)torrent["created_at"]);
-
+                        publishDate = DateTime.Parse((string)torrent["created_at"]);
+                    Uri bannerUrl = null;
                     if (torrent["portada"] != null)
-                        release.BannerUrl = new Uri(BannerUrl + (string)(torrent["portada"]["hash"]) + "." + (string)(torrent["portada"]["ext"]));
-
-                    release.Category = MapTrackerCatToNewznab((string)torrent["categoria"]);
-                    release.Size = (long)torrent["size"];
-
-                    release.Seeders = (int)torrent["seeders"];
-                    release.Peers = release.Seeders + (int)torrent["leechers"];
-                    release.Grabs = (long)torrent["snatched"];
+	                    bannerUrl = new Uri(BannerUrl + (string)(torrent["portada"]["hash"]) + "." + (string)(torrent["portada"]["ext"]));
 
-                    release.InfoHash = (string)torrent["plain_info_hash"];
-                    release.Link = new Uri(DownloadUrl + (string)torrent["id"]);
+                    var seeders = (int)torrent["seeders"];
 
-                    var files = (JArray)JsonConvert.DeserializeObject<dynamic>((string)torrent["files_list"]);
-                    release.Files = files.Count;
-
-                    release.DownloadVolumeFactor = (string)torrent["freetorrent"] == "0" ? 1 : 0;
-                    release.UploadVolumeFactor = (string)torrent["doubletorrent"] == "0" ? 1 : 2;
-
-                    release.MinimumRatio = 1;
-                    release.MinimumSeedTime = 172800; // 48 hours
-
-                    releases.Add(release);
+                    releases.Add(new ReleaseInfo
+                    {
+                        Title = title,
+                        Category = MapTrackerCatToNewznab((string)torrent["categoria"]),
+                        Size = (long)torrent["size"],
+                        Grabs = (long)torrent["snatched"],
+                        InfoHash = (string)torrent["plain_info_hash"],
+                        Link = new Uri(DownloadUrl + (string)torrent["id"]),
+                        Files = ((JArray)JsonConvert.DeserializeObject<dynamic>((string)torrent["files_list"])).Count,
```

Comment:
As I said in other comment, make the conversions that can fail outside the initializer.